src/index.css

```diff
@@ -408,3 +408,74 @@ hr.horizontalLine.opm {
 hr.horizontalLine.opmComparison {
   background: linear-gradient(to right, #6370FF, #5A9EF500);
 }
+
+/* Order Details */
+#orderDetailsComp{
+  @apply font-helvetica;
+}
+#orderDetailsComp .grid span {
+  @apply text-sm py-2 text-[#FAF9F6] bg-[#30343B];
```

Comment:
Any reason why u added this here, instead of directly adding it in the span element in `OrderDetails.tsx` ?

Reply:
#orderDetailsComp .grid span style is only taking place for the cards that show info except SKU item info(Table). The table th and td also contain the span.  I think this is the safest way to style those. We don't need to add it in all span elements. As of now this is working as expected.